frontend/ui/widget/menu.lua

```diff
@@ -899,6 +899,9 @@ function Menu:updatePageInfo(select_number)
         end
         -- update page information
         self.page_info_text:setText(util.template(_("page %1 of %2"), self.page, self.page_num))
+        if self.page == self.page_num then
+          self.page_info_text:setText("");
```

Comment:
In file browser, when on the last page, you'd see blank (between arrow buttons) instead of `Page 3 of 3`.
Probably better that way:
```lua
if self.page_num > 1 then
    self.page_info_text:setText(util.template(_("page %1 of %2"), self.page, self.page_num))
else
    self.page_info_text:setText("");
end
```

Reply:
Absolutely agree, will update my code now, didn't think of that!